phpstan.neon.dist

```diff
@@ -46,6 +46,7 @@ parameters:
     - '#constructor expects Symfony\\Contracts\\Translation\\TranslatorInterface\|null, Symfony\\Component\\Translation\\TranslatorInterface given\.$#'
     - '#^Method Symfony\\Contracts\\EventDispatcher\\EventDispatcherInterface\:\:dispatch\(\) invoked with 2 parameters, 1 required\.$#'
     - '~^Access to an undefined property PrestaShop\\PrestaShop\\Core\\Module\\ModuleInterface::\$(attributes|disk|database)\.$~'
+    - '#^Readonly property .+ is assigned outside of the constructor\.$#'
```

Comment:
not sure if this is good way, but else stan fails in legit cases like https://github.com/PrestaShop/PrestaShop/pull/32464/files#diff-1e107577cb66de89cf86fcdebe848a2e33a82f3cceb7b081b30c5f6931c9fd0cR61

Reply:
PHPStan is right here, the code will fail because the property is read only and the code is updating the value